sensitivity.tex

```diff
@@ -0,0 +1,126 @@
+\section{Sensitivity Analysis}
+
+The most important aspect of any fuel cycle transition scenario
+is the accrual of fissile materials for new reactor deployment.
+The collaborative strategy makes a transition possible materialistically,
+but the aggressive transition demands an increase in reprocessing capacity.
+This is due to both high initial plutonium loading of the \gls{ASTRID}
+reactors and the low plutonium composition in spent \gls{UOX} fuel.
+
+We varied two parameters, the lifetime of French \glspl{LWR} and the
+breeding ratio of \gls{ASTRID} reactors. The parameters and their values
+are listed in \cref{tab:sen_par}
+
+\begin{table}[h]
+    \centering
+    \begin{tabularx}{\textwidth}{lbb}
+        \hline
+        \textbf{Parameter} & \textbf{Original} & \textbf{Values} \\
+        \hline
+        Lifetime of French \glspl{LWR} [years] & 60  & 65, 70, 80 \\
+        Breeding Ratio of \glspl{ASTRID} & 1.08 & 1.11, 1.15, 1.18 \\ 
+        \hline
+    \end{tabularx}
+    \caption {Two parameters are varied for the sensitivity analysis - the 
```

Comment:
This should be past tense. Tense may feel tricky, but it's really not. A good rule is to write tense truthfully (e.g. try not to write that something is happening or will happen if it has already happened.)